x/session/keeper/session_hydrator.go

```diff
@@ -266,10 +267,40 @@ func sha3Hash(bz []byte) []byte {
 	return hasher.Sum(nil)
 }
 
+// GetSessionId returns the string and bytes representation of the sessionId
+// given the application public key, service ID, block hash, and block height
+// that is used to get the session start block height.
+func (k Keeper) GetSessionId(
+	ctx context.Context,
+	appPubKey,
+	serviceId string,
+	blockHashBz []byte,
+	blockHeight int64,
+) (sessionId string, sessionIdBz []byte) {
+	sharedParams := k.sharedKeeper.GetParams(ctx)
+	return GetSessionId(&sharedParams, appPubKey, serviceId, blockHashBz, blockHeight)
+}
+
+// GetDefaultSessionId returns the string and bytes representation of the sessionId
+// given the application public key, service ID, block hash, and block height
+// that is used to get the session start block height.
+//
+// TODO_TECHDEBT(#517): Move this to a shared testutil.
+func GetDefaultSessionId(
```

Comment:
Same comment as on the other PRs related to where `Default` should go in the naming scheme.